[2023-03-23 15:16:51,719] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_defence_dag.Defence2 manual__2023-03-23T15:16:30.344788+00:00 [queued]>
[2023-03-23 15:16:51,803] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_defence_dag.Defence2 manual__2023-03-23T15:16:30.344788+00:00 [queued]>
[2023-03-23 15:16:51,805] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-23 15:16:51,807] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-03-23 15:16:51,809] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-23 15:16:51,899] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): Defence2> on 2023-03-23 15:16:30.344788+00:00
[2023-03-23 15:16:51,925] {standard_task_runner.py:52} INFO - Started process 11887 to run task
[2023-03-23 15:16:51,973] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'multi_defence_dag', 'Defence2', 'manual__2023-03-23T15:16:30.344788+00:00', '--job-id', '2956', '--raw', '--subdir', 'DAGS_FOLDER/defence_dag.py', '--cfg-path', '/tmp/tmplbb46p60', '--error-file', '/tmp/tmpsc4vh6kg']
[2023-03-23 15:16:51,983] {standard_task_runner.py:80} INFO - Job 2956: Subtask Defence2
[2023-03-23 15:16:52,508] {task_command.py:369} INFO - Running <TaskInstance: multi_defence_dag.Defence2 manual__2023-03-23T15:16:30.344788+00:00 [running]> on host 6447f1dacd32
[2023-03-23 15:16:53,135] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=multi_defence_dag
AIRFLOW_CTX_TASK_ID=Defence2
AIRFLOW_CTX_EXECUTION_DATE=2023-03-23T15:16:30.344788+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-23T15:16:30.344788+00:00
[2023-03-23 15:16:53,429] {python.py:173} INFO - Done. Returned value was: None
[2023-03-23 15:16:53,681] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=multi_defence_dag, task_id=Defence2, execution_date=20230323T151630, start_date=20230323T151651, end_date=20230323T151653
[2023-03-23 15:16:53,946] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-23 15:16:54,416] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
